src/ringcentral-web-phone.js

```diff
@@ -206,6 +206,11 @@
             this.userAgent.audioHelper.playIncoming(true);
             patchSession(session);
             patchIncomingSession(session);
+            session.sendReceiveConfirm().then(function() {
+                session.logger.log('sendReceiveConfirm success');
+            }).catch(function(error){
+                console.error('failed to send receive confirmation via SIP MESSAGE due to ' + error);
```

Comment:
Console output is forbidden in library code. You may use session logger if needed, but you have to also emit an event. Please talk to @vyshakhbabji as he did some research about it.

Reply:
OK, I will update it.